.github/workflows/file-size-impact/report-file-size-impact.js

```diff
@@ -0,0 +1,14 @@
+const { reportFileSizeImpact, readGithubWorkflowEnv } = require("@jsenv/file-size-impact");
+
+reportFileSizeImpact({
+  ...readGithubWorkflowEnv(),
+  projectDirectoryUrl: process.env.GITHUB_WORKSPACE,
```

Comment:
`projectDirectoryUrl` is already provided by `readGithubWorkflowEnv` in https://github.com/jsenv/jsenv-file-size-impact/blob/4b1e4f5e0c7f78e0b0caf3042f914a0a8e1d6fcc/src/readGithubWorkflowEnv.js#L35

You can remove this line and keeping it is fine too, as you prefer.

Reply:
Right!
I just removed all the properties that were replicating the default behavior.